engine/Scripts/lib_build_android.sh

```diff
@@ -1,7 +1,7 @@
 engine_path=$(pwd)
 runtime_mode=release
-
-while getopts ":r:m:" opt
+architecture=32
```

Comment:
32 is not a proper default value for architecture. It might be fine to just leave it empty (i.e., architecture = "") when no value is specified.

One more thing, we should also check the input arguments, so user can get some warnings if the argument is not valid (e.g., sh lib_build_android.sh -a win32)